aten/src/ATen/native/SpectralOps.cpp

```diff
@@ -924,17 +924,20 @@ Tensor istft(const Tensor& self, const int64_t n_fft, const optional<int64_t> ho
 
   y = y.slice(2, start, end, 1);
   window_envelop = window_envelop.slice(2, start, end, 1);
-  const auto window_envelop_lowest = window_envelop.abs().min().item().toDouble();
-  if (window_envelop_lowest < 1e-11) {
-    std::ostringstream ss;
-    REPR(ss) << "window overlap add min: " << window_envelop_lowest;
-    AT_ERROR(ss.str());
-  }
 
-  y = (y / window_envelop).squeeze(1);  // size: (channel, expected_output_signal_len)
+  // Apply window normalization on non-tiny indices
+  window_envelop = at::where(
+    window_envelop.abs() > 1e-11, window_envelop, at::ones({}, window_envelop.options()));
+  y = y / window_envelop;
+  y = y.squeeze(1);  // size: (channel, expected_output_signal_len)
   if (input_dim == 3) {
     y = y.squeeze(0);
   }
+
+  // zero padding if the given lengthOpt is longer than expected
+  if(end > expected_output_signal_len) {
```

Comment:
Can you update the docs to reflect this behavior, and write a note that high `hop_length` causes gaps in the the inversed signal.